tests/main/retry-network/task.yaml

```diff
@@ -0,0 +1,43 @@
+summary: Test httputil/retry.go ShouldRetryError(err)/NoNetwork(err)
+
+# ubuntu-core: no go-compiler
+# ubuntu-14.04: no nsenter
+systems: [-ubuntu-core-*, -ubuntu-14.04-*]
+
+environment:
+    # an empty $topsrcdir/tests/go.mod seems to break importing or building go
+    # packages referenced by their import paths while under the tests directory,
+    # need to disable go modules supportfor this test
```

Comment:
Other ways to handle this situation would be to just to add a minimal go.mod or use the go snap and install a Go version that doesn't care about go modules.